plugins/system/apticron

```diff
@@ -0,0 +1,228 @@
+     Language=${language:-en}
+
+     Warning_Update=${warning_update:-$Warning}
+     Warning_Newone=${warning_newone:-$((Warning_Update/4+1))}
+     Warning_Remove=${warning_remove:-$((Warning_Update/50+1))}
+     Warning_Holdit=${warning_holdit:-$((Warning_Update/50+1))}
+
+     Critical_Update=${critical_update:-$Critical}
+     Critical_Newone=${critical_newone:-$((Critical_Update/4+1))}
+     Critical_Remove=${critical_remove:-$((Critical_Update/50+1))}
+     Critical_Holdit=${critical_holdit:-$((Critical_Update/50+1))}
+
+     # Localisation of the graphic texts 
+     case $Language in
+          de)
+            echo "graph_title     Ausstehende Aktualisierungen"
+            echo "graph_vlabel    Anzahl Pakete"
+            echo "graph_info      Die Grafik zeigt die Ausgabe von: apt-get -u dist-upgrade."
+            update_label="aktualisieren"
+            newone_label="neu installieren"
+            remove_label="entfernen"
+            holdit_label="nicht aktualisieren"
+            ;;
+          es) 
+            echo "graph_title     Actualizaciones pendientes " 
+            echo "graph_vlabel    Número de paquetes de software" 
+            echo "graph_info      El gráfico muestra el resultado de: apt-get -u dist-upgrade. "
+            update_label="actualización"
+            newone_label="reinstalar "
+            remove_label="eliminar"
+            holdit_label="no actualizar"
+            ;;
+           *)
+            echo "graph_title     Pending updates"
+            echo "graph_vlabel    Count of packages"
+            echo "graph_info      The graph shows the optput of: apt-get -u dist-upgrade."
+            update_label="upgraded"
+            newone_label="newly installed"
+            remove_label="to remove"
+            holdit_label="not upgraded"
+            ;;
+     esac
+
+
+     echo "graph_args      --base 1000 -l 100"
+     echo "graph_scale     no"
+     echo "graph_category  system"
+
+     echo "update.label    $update_label"
+     echo "newone.label    $newone_label"
+     echo "remove.label    $remove_label"
+     echo "holdit.label    $holdit_label"
+
+     echo "update.warning  $Warning_Update"
+     echo "newone.warning  $Warning_Newone"
+     echo "remove.warning  $Warning_Remove"
+     echo "holdit.warning  $Warning_Holdit"
+
+     echo "update.critical $Critical_Update"
+     echo "newone.critical $Critical_Newone"
+     echo "remove.critical $Critical_Remove"
+     echo "holdit.critical $Critical_Holdit"
+
+     exit 0
+  fi
+
+###################################################
+# Data reading sections                           #
+###################################################
+
+# call apt-get command and sepate the results
+  AptInfo=$(apt-get -u dist-upgrade --print-uris --yes | grep upgraded.)
+  Ausgabe=($(echo "${AptInfo}" | awk 'match($0,/([0-9]+) upgraded, ([0-9]+) newly installed, ([0-9]+) to remove and ([0-9]+) not upgraded./,arr){print arr[1], arr[2], arr[3], arr[4]};'))
```

Comment:
This may work with the current version of apt-get. But it will fail whenever there is the slightest change in the status output. `apt-get` is an interactive program. Its output is targeted at humans and thus should not be parsed.

Please look for another way to get these numbers.